### PR TITLE
Add missing descriptions for CLI commands

### PR DESCRIPTION
**Pull request by piotr-iohk**
# Issue Number

# Overview

- [ ] Added descriptions for commands that were missing it in CLI help

# Comments

Before:

```

Available options:
  -h,--help                Show this help text

Available commands:
  launch                   Launch and monitor a wallet server and its chain
                           producers.
  serve                    serve API that listens for commands/actions.
  mnemonic                 
  wallet                  
  transaction             
  address                  
  version                  Show the program's version.

```

After:

```
Available options:
  -h,--help                Show this help text

Available commands:
  launch                   Launch and monitor a wallet server and its chain
                           producers.
  serve                    Serve API that listens for commands/actions.
  mnemonic                 Manage mnemonic phrases.
  wallet                   Manage wallets.
  transaction              Manage transactions.
  address                  Manage addresses.
  version                  Show the program's version.

```

### CODE REVIEW

**Comment by KtorZ** [comment]
@piotr-iohk What about "Tooling around mnemonic words" instead of "Generate mnemonic words." ? Since we have a `generate` command, it suggests that there could be more (and there might be). So, `mnemonic` is really just there to regroup commands about mnemonic.

**Comment by piotr-iohk** [comment]
Rephrased to "Tooling around mnemonic words".

**Review by KtorZ — APPROVED** [review]

**Review by paweljakubas — APPROVED** [review]

File: lib/cli/test/unit/Cardano/CLISpec.hs [hunk 1]
```diff
@@ -111,14 +111,15 @@ spec = do
             , "  -h,--help                Show this help text"
             , ""
             , "Available commands:"
-            , "  mnemonic                 "
-            , "  wallet                   "
-            , "  transaction              "
-            , "  address                  "
+            , "  mnemonic                 Tooling around mnemonic words."
```
Comment (jonathanknowles) [inline lib/cli/test/unit/Cardano/CLISpec.hs]: This phrasing seems a bit inconsistent.

The other descriptions are all **actions** that one can perform:

* "Manage wallets"
* "Manage transactions"
* "Manage addresses"

But "Tooling around ... " is not an action, which makes it inconsistent.

How about:

* "Generate mnemonics."

**Review by jonathanknowles — CHANGES_REQUESTED** [review]
I think this is a good addition!

However, I think the following styles are a bit inconsistent:

> 1. "Manage wallets"
> 2. "Manage transactions"
> 3. "Manage addresses"
> 4. "Tooling around mnemonic words"

The first 3 are **actions** that one can perform. But the last one is a **noun phrase**, not an action, which makes it feel inconsistent.

## Solution A

How about:

* "Generate mnemonic phrases."
* "Manage mnemonic phrases."

Any of these would be consistent with the style you have used for "wallet", "transaction" and "address".

## Solution B

Alternatively, make the whole lot into **noun phrases**:

* "Commands relating to mnemonic phrases and their generation."
* "Commands relating to transactions."
* "Commands relating to wallets."
* "Commands relating to addresses."

File: lib/cli/test/unit/Cardano/CLISpec.hs (hunk 1 above)
Comment (piotr-iohk) [reply lib/cli/test/unit/Cardano/CLISpec.hs]: Originally it was `Generate mnemonic words.` but as pointed [here](https://github.com/input-output-hk/cardano-wallet/pull/690#issuecomment-529455047) it doesn't have to be only about generating, so maybe let's go for `Manage mnemonic phrases.`.

Comment (jonathanknowles) [reply lib/cli/test/unit/Cardano/CLISpec.hs]: That sounds like a fair compromise!

**Review by jonathanknowles — APPROVED** [review]

**Pull request merged**